openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +488,39 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is offered at the Request URI endpoint by the Verifier. In case of success, the response is a Request Object that the Wallet MUST process in the same way as a Request Object as defined in [@RFC9101]. 
+
+The request MUST use the HTTP POST method with the https scheme and the media type set to "application/oauth-authz-req+jwt".
+
+The following parameters are defined: 
+
+`state`:
+: OPTIONAL. A JSON String containing the value of the corresponding authorization Request's `state` parameter, if present.
+
+`issuer`:
+: OPTIONAL. A JSON containing an HTTPS URL designating the Issuer URL of the Wallet (acting as a OAuth 2.0 Authorization Server). The Verifier MAY obtain the Wallet's metadata by adding the well-know location `oauth-authorization-server` as specified in [@!RFC8414]. Metadata MAY also be provided by other means, for example in the `wallet_metadata`parameter`.
+
+`wallet_metadata`:
+: OPTIONAL. A JSON Object containing metadata parameters as defined in (#as_metadata_parameters).
+
+`issuer_nonce`:
+: OPTIONAL. A JSON String containing as fresh, cryptographically random number with sufficient entropy the Verifier MUST use when creating the signed presentation request object. 
+
+`w_ephm_key`:
```

Comment:
I would add the following:
- If `wallet_metadata` is present, then the `issuer` parameter MUST be omitted.
- I would rename `w_ephm_key` to `wallet_epk`. 

Ideally, we would also use `jwks` from the `wallet_metadata` instead of `w_ephm_key` but I just figured that AS metadata only defines `jwks_uri`. `jwks` is only supported for clients.

Reply:
I just checked and OpenID Federation actually uses `jwks` but I couldn't find it in the IANA registry yet, so reusing `jwks` from `wallet_metadata` would still be an option. In that case, we should say that `wallet_epk` MUST not be present if `wallet_metadata` is present, or MUST match the `jwks` in the metadata.

Reply:
Why do we need an issuer of the wallet at all? This only risks fragmentation of the ecosystem. I would keep wallet_metadata as anonymous and general as possible.

Reply:
I'm fine with moving the key into the `wallet_metadata` as this mirrors the way we do it for verifiers. We can register the `jwk` claim in our spec.